golang/cosmos/x/swingset/keeper/keeper.go

```diff
@@ -442,8 +442,8 @@ func (k Keeper) SwingStore(ctx sdk.Context) types.Store {
 	return types.NewStore(store, []byte(swingStoreKeyPrefix))
 }
 
-func (k Keeper) ExportSwingStore(ctx sdk.Context) []*vstoragetypes.DataEntry {
-	return k.vstorageKeeper.ExportStorageFromPrefix(ctx, StoragePathSwingStore)
+func (k Keeper) ExportSwingStore(ctx sdk.Context) sdk.Iterator {
```

Comment:
Add godoc comments for all exported symbols:

```
// ExportSwingStore returns an iterator for the SwingStore.
```

and consider renaming to `GetSwingStoreIterator()`.

Or even better, replace all calls with the body on the next line. I'm not sure if this method is a useful abstraction.